eng/pipelines/performance/templates/build-perf-sample-apps.yml

```diff
@@ -1,35 +1,21 @@
 parameters:
   osGroup: ''
-  osSubgroup: ''
-  archType: ''
-  buildConfig: ''
-  runtimeFlavor: ''
-  helixQueues: ''
-  targetRid: ''
+  flavor: 'mono' # Currently only used for Android Hello World app
```

Comment:
should we call it `runtimeFlavor` for consistency with other yaml files and scripts?

Reply:
I initially started out with it being runtimeFlavor but ended up hitting an error in the yaml as it collided with a separately set runtimeFlavor. In my testing, this colliding runtimeFlavor didn't seem to get properly passed through to the build-perf-sample-apps either. I will add a comment about that.

Reply:
Thinking about it some more and looking at some deeper parts of the pipeline, we end up saving what is essentially this value in RuntimeType. Would it make sense to use that as the name here to make a clear connection between the two? At a high level, I think it seems like an alright name substitution but maybe there is some context I am missing.

Reply:
Thanks for documenting this. I am bit apprehensive about the fact that we have `flavor` and `runtimeFlavor`. Is something that we can fix long term or the cost would be too big?